### PR TITLE
Feat noticias

### PR DESCRIPTION
**Pull request by mixcas**

### CODE REVIEW

File: src/components/NoticiaForm.jsx [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+  constructor(props) {
+    super(props);
+
+    // If component recieves noticia as prop we merge it with initial state (used for editing)
+    this.state = { ...this.state, ...props.noticia };
+
+  }
+
+  componentDidMunt
```
Comment (mikeyrayvon) [inline src/components/NoticiaForm.jsx]: this can go?

File: src/components/NoticiaForm.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+  }
+
+  updateNoticia() {
+    const { title, published } = this.state;
+
+    this.setState({ isLoading: true })
+
+    this.props.firebase
+      .update(`noticias/${this.props.id}`, {
```
Comment (mikeyrayvon) [inline src/components/NoticiaForm.jsx:96]: why backticks?

File: src/containers/Noticias.jsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { compose } from 'redux';
+import { connect } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { firebaseConnect } from 'react-redux-firebase';
+
+import NoticiasList from '../components/NoticiasList';
+
+const Noticias = ({ firebase, noticias }) => (
```
Comment (mikeyrayvon) [inline src/containers/Noticias.jsx]: `firebase` isn't needed here

File: src/components/NoticiasList.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+            (key, id) => <NoticiasListItem key={key} id={id} noticia={noticias[key]} />
+          ) }
+        </div>
+      </section>
+    );
+  }
+
+};
+
+export default withRouter(NoticiasList);
```
Comment (varffs) [inline src/components/NoticiasList.jsx]: Be good to confirm if/why this is `withRouter`

File: src/components/NoticiasListItem.jsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <h3><Link to={'/noticias/' + key}>{title}</Link></h3>
+      </div>
+      <div className='grid-item item-s-1'>
+        <h3>{published ? 'Publicado' : 'Borrador'}</h3>
+      </div>
+      <div className='grid-item item-s-1 offset-s-5'>
+        <p><Link to={'/noticias/' + key}>Editar</Link></p>
+      </div>
+      <div className='grid-item item-s-1'>
+        <button onClick={() => window.confirm('¿Seguro que deseas eliminar esta noticia?') ? remove('noticias/' + key) : null}>Eliminar</button>
```
Comment (varffs) [inline src/components/NoticiasListItem.jsx:22]: is `remove()` a function from firebase here?

File: src/containers/Noticias.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+  <section>
+
+    <div className='grid-row'>
+      <div className='grid-item item-s-3'>
+        <h2>Noticias</h2>
```
Comment (varffs) [inline src/containers/Noticias.jsx:14]: These kind of list view layouts are going to be quite common. Can we make this flexible for all post types?

File: src/containers/UpdateNoticia.jsx [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+import NoticiaForm  from '../components/NoticiaForm.jsx';
+
+const UpdateNoticia = ({ firebase, noticia, id }) => {
+
+  if (!isLoaded(noticia)) { // If not loaded…
+    return 'Loading'; // …show 'loading'
+  } else if (isEmpty(noticia)) { // …else. If is empty…
+    return 'Error'; // …show 'Error?'
+  } else {
+      console.log(id);
```
Comment (varffs) [inline src/containers/UpdateNoticia.jsx]: ooo :)

**Review by varffs — COMMENTED** [review]
Add all fields as simple inputs to the form and this should be done.

File: src/components/NoticiaForm.jsx (hunk 2 above)
Comment (mixcas) [reply src/components/NoticiaForm.jsx:96]: its some new shiny ES6 string composition, see the variable  inside `${}`? 
[;

File: src/components/NoticiasListItem.jsx (hunk 5 above)
Comment (mixcas) [reply src/components/NoticiasListItem.jsx:22]: Yes, `(firebase: { remove }) => {}` means:
```
const firebase = this.props.firebase;
const remove = firebase.remove; // or this.props.firebase.remove
```
Although I will change it so we use full `firebase.remove` so its readable.

File: src/containers/Noticias.jsx (hunk 6 above)
Comment (mixcas) [reply src/containers/Noticias.jsx:14]: Sure, once we get 2-3 other pages would be good to find an abstraction.

**Comment by mixcas** [comment]
This PR is ready. @mikeyrayvon already went thru it when adding WYSIWYG. Plz check and close @varffs.

**Pull request merged**